pkg/errors/types.go

```diff
@@ -42,4 +42,10 @@ var (
 
 	// ErrScanMetadata indicates problem with metadata in db
 	ErrScanMetadata = New("failed to scan metadata in db")
+
+	// ErrFailedFetch indicates that fetching of entity data failed.
+	ErrFailedFetch = New("failed to fetch entity")
+
+	// ErrFailedConnect indicates that connecting thing to channel failed.
+	ErrFailedConnect = New("failed to connect thing to channel")
```

Comment:
Are these errors used? I see them used in the context of `errors.Contains` but they are never returned (directly or as a wrap), so those checks make no sense. Please double-check.

Reply:
Yes, it actually was useless to keep these errors. I've removed these two.